deployments/kustomize/cronjobs/BACKUP-update-current-year-cron.yaml

```diff
@@ -0,0 +1,216 @@
+spec:
+  schedule: "*/5  * * * *"
+  concurrencyPolicy: Forbid
+  suspend: false
+  jobTemplate:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: gdx-agreements-tracker
+        cronjob: postgres-cron-current-year
+        env: dev
+        role: pmo-update-current-year
+        template: postgres-cron-current-year-cronjob
+    spec:
+      template:
+        metadata:
+          creationTimestamp: null
+          labels:
+            app: gdx-agreements-tracker
+            cronjob: postgres-cron-current-year
+            env: dev
+            role: pmo-update-current-year
+            template: postgres-cron-current-year-cronjob
+        spec:
+          restartPolicy: Never
+          activeDeadlineSeconds: 1600
+          serviceAccountName: default
+          schedulerName: default-scheduler
+          terminationGracePeriodSeconds: 30
+          securityContext: {}
+          containers:
+            - name: postgres-current-year-updater
+              image: >-
+                image-registry.openshift-image-registry.svc:5000/acd38d-tools/gdx-agreements-tracker-backup-container:latest
+              command:
+                - /bin/bash
+                - "-c"
+                - >
+                  psql
+                  "postgresql://$DATABASE_USER:$DATABASE_PASSWORD@$DATABASE_HOST/$DATABASE_NAME"
+                  <<EOF
+
+                  DO \$\$ DECLARE
+
+                      current_fiscal VARCHAR(5);
+                      current_year INTEGER;
+                      next_year INTEGER;
+                      next_fiscal VARCHAR(5);
+                  BEGIN
+
+                      -- Get the current fiscal year
+                      SELECT fiscal_year INTO current_fiscal FROM data.fiscal_year WHERE is_current = true;
+
+                      -- Calculate the next fiscal year
+                      current_year := CAST(SPLIT_PART(current_fiscal, '-', 2) AS INTEGER);
+                      next_year := current_year + 1;
+                      next_fiscal := CONCAT(RIGHT(current_year::TEXT, 2), '-', RIGHT(next_year::TEXT, 2));
+
+                      -- Update is_current for the next fiscal year
+                      UPDATE data.fiscal_year SET is_current = false;
+                      UPDATE data.fiscal_year SET is_current = true WHERE fiscal_year = next_fiscal;
+
+                      -- TESTING ONLY! Set the fiscal_year back to '22-23' again (testing purposes only)
+                      IF NOT EXISTS (SELECT 1 FROM data.fiscal_year WHERE is_current = true) THEN
+                      UPDATE data.fiscal_year SET is_current=true WHERE fiscal_year='22-23';
+                      END IF;
+                  END \$\$;
+
+                  SELECT * FROM data.fiscal_year ORDER BY id ASC;
+
+                  EOF
+              env:
+                - name: DATABASE_HOST
+                  valueFrom:
+                    configMapKeyRef:
+                      name: pmo-config-ddg76ttmch
```

Comment:
these should not be hardcoded, kustomize creates the -ddg76ttmch, you should just have config